pydnstest/scenario.py

```diff
@@ -319,13 +327,16 @@ def set_edns(self, fields):
                 subnet_addr = net[0]
                 addr = socket.inet_pton(family, net[0])
                 prefix = len(addr) * 8
+                scope = 0
                 if len(net) > 1:
                     prefix = int(net[1])
+                if len(net) > 2:
```

Comment:
I have no experience with ECS so it is hard to review without documentation. Please open a new MR on Gitlab and I will review it there. Thank you for your time!